ERCS/erc-7741.md

```diff
@@ -0,0 +1,211 @@
+---
+eip: 7741
+title: Authorize Operator
+description: Set Operator via EIP-712 secp256k1 signatures
```

Comment:
Would be nice to expand on the title a bit more. You have 140-ish characters for a description, so you should use them. Maybe define what an "Operator" is?